network/core-node/admin-guide/monitoring.mdx

```diff
@@ -125,34 +142,143 @@ This list is the result of both inbound connections from other peers and outboun
+
+- `backlog`: List of nodes for which the survey request are yet to be sent
+- `badResponseNodes`: List of nodes that sent a malformed response
+- `topology`: Map of nodes to connection information
+  - `inboundPeers`/`outboundPeers`: List of connection information by nodes
+    - `bytesRead`: The total number of bytes read from this peer.
+    - `bytesWritten`: The total number of bytes written to this peer.
+    - `duplicateFetchBytesRecv`: The number of bytes received that were duplicate transaction sets and quorum sets.
+    - `duplicateFetchMessageRecv`: The count of duplicate transaction sets and quorum sets received from this peer.
+    - `duplicateFloodBytesRecv`: The number of bytes received that were transactions and SCP votes duplicates.
+    - `duplicateFloodMessageRecv`: The count of duplicate transactions and SCP votes received from this peer.
+    - `messagesRead`: The total number of messages read from this peer.
+    - `messagesWritten`: The total number of messages written to this peer.
+    - `nodeId`: Node's public key.
+    - `secondsConnected`: The total number of seconds this peer has been connected to the surveyed node.
+    - `uniqueFetchBytesRecv`: The number of bytes received that were unique transaction sets and quorum sets.
+    - `uniqueFetchMessageRecv`: The count of unique transaction sets and quorum sets received from this peer.
+    - `uniqueFloodBytesRecv`: The number of bytes received that were unique transactions and SCP votes.
+    - `uniqueFloodMessageRecv`: The count of unique transactions and SCP votes received from this peer.
+    - `version`: stellar-core version.
+- `numTotalInboundPeers`/`numTotalOutboundPeers`: The number of total inbound and outbound peers this node is connected to. The response will have a random subset of 25 connected peers per direction (inbound/outbound). These fields tell you if you're missing nodes so you can send another request out to get another random subset of nodes.
+- `maxInboundPeerCount`/`maxOutboundPeerCount`: The number of total inbound and outbound peers that this node can accept. These fields correspond to stellar-core configurations `MAX_ADDITIONAL_PEER_CONNECTIONS` and `TARGET_PEER_CONNECTIONS`, respectively.
 
 ## Quorum Health
 
 To help node operators monitor their quorum sets and maintain the health of the overall network, Stellar Core also provides metrics on other nodes in your quorum set. You should monitor them to make sure they're up and running, and that your quorum set is maintaining good overlap with the rest of the network.
 
-### Quorum set diagnostics
+### Quorum Set Diagnostics
 
-The `quorum` command allows to diagnose problems with the quorum set of the local node.
+The `quorum` command allows you to diagnose problems with the quorum set of the local node.
 
 If you run:
 
-`$ stellar-core http-command 'quorum'`
+```bash
+stellar-core http-command 'quorum'
+```
 
 The output will look something like:
 
```

Comment:
If you want to see how it looks right now, here is an [example](https://gist.github.com/anupsdf/7480bab592bf9d0df686674928cfdd98). Up to you if you want to keep this one since its short.

Reply:
~~Yeah, I think I'll leave it as-is, just for the sake of brevity. The page is already pretty long, and the more fully-fleshed-out will probably add to that fact quite a bit...~~

On second thought, it appears there are some substantial differences in the JSON object that is output. I think I'll copy/paste but modify/truncate the `cost` objects to save some room.

Reply:
On third thought, nevermind. It seems like even truncating it to three validators is still tremendously long, and some of the explanatory text goes along with the specific example that is listed here.